tasks/58_missed-letters/missed-letters.js

```diff
@@ -0,0 +1,11 @@
+function findLetter(str) {
+  const lets = str.split('');
```

Comment:
Why you save letters? Are you afraid to run out of them? `letters` is a much better readable name.